src/main/java/org/broadinstitute/hellbender/utils/read/markduplicates/OpticalDuplicateFinder.java

```diff
@@ -185,6 +188,14 @@ protected final int rapidParseInt(final String input) {
         return val;
     }
 
+    public boolean isOpticalDuplicate(PhysicalLocation a, PhysicalLocation b) {
+      return a.getReadGroup() == b.getReadGroup() &&
+          a.getTile() == b.getTile() &&
+          // There used to not be an abs here, not sure why.
```

Comment:
I would guess that the only method calling was iterating through these pairs in sorted order, such that ```a.getX() => b.getX()``` and ```a.getY() => b.getY()```.   See [line 213](https://github.com/broadinstitute/hellbender/pull/587/files#diff-713d4daca4858162296833c52630ae1fR213) (in ```findOpticalDuplicates```).